uint64 pose_id
Keypoint[] points